core/templates/dev/head/pages/donate-page/donate-page.mainpage.html

```diff
@@ -1,39 +1,94 @@
+            <span class="oppia-navbar-breadcrumb-separator"></span>
+            Donate
+          </li>
+        </ul>
+      </navbar-breadcrumb>
 
-{% block navbar_breadcrumb %}
-  <ul class="nav navbar-nav oppia-navbar-breadcrumb">
-    <li>
-      <span class="oppia-navbar-breadcrumb-separator"></span>
-      Donate
-    </li>
-  </ul>
-{% endblock navbar_breadcrumb %}
-
-{% block content %}
-  <donate-page></donate-page>
-{% endblock %}
-
-{% block footer_js %}
-  {{ super() }}
-  <!-- This code is used for inserting webpack bundles
-     https://github.com/jantimon/html-webpack-plugin#writing-your-own-templates -->
-  <% for (var chunk in htmlWebpackPlugin.files.js) { %>
-    <% if (webpackConfig.mode == 'production') { %>
-      <script src="/build/templates/head/dist/<%= htmlWebpackPlugin.files.js[chunk] %>"></script>
-    <% } else { %>
-      <script src="/dist/<%= htmlWebpackPlugin.files.js[chunk] %>"></script>
-    <% } %>
-  <% } %>
-{% endblock footer_js %}
-
-{% block footer %}
-  <oppia-footer></oppia-footer>
-{% endblock %}
+      <content>
+        <donate-page></donate-page>
+      </content>
+    </base-content>
+    <style>
+      h1.oppia-donate-h1 {
+        color: #fff;
+        font-size: 1.5em;
+        margin-top: 60px;
+        text-align: center;
+        line-height: 1.7em;
+      }
+      h3.oppia-donate-h3 {
+        font-size: .75em;
+        font-weight: normal;
+        margin: 2.33em 0;
+        text-align: center;
+      }
+      .oppia-donation-card-content-wide {
+        margin: 40px auto 80px;
+        min-height: 670px;
+        overflow: auto;
+      }
+      .oppia-donate-options {
+        background-color: #009688;
+        display: block;
+        float: right;
+        font-family: "Capriola", "Roboto", Arial, sans-serif;
+        padding: 10px 10px 35px 10px;
+        width: 340px;
+      }
+      .btn.oppia-donate-options-button {
+        background-color: rgba(0,0,0,0.2);
+        color: rgba(255,255,255,1.0);
+        font-family: "Capriola", "Roboto", Arial, sans-serif;
+        font-size: 16px;
+        height: 40px;
+        margin: 10px auto;
+        width: 180px;
+      }
+      .btn.oppia-donate-options-button:hover,
+      .btn.oppia-donate-options-button:focus {
+        background-color: rgba(5, 190, 178, 1);
+        color: rgba(255,255,255,1);
+      }
+      .oppia-donate-info {
+        float: left;
+        padding-bottom: 30px;
+        width: -webkit-calc(100% - 340px);
+        width: -moz-calc(100% - 340px);
+        width: -o-calc(100% - 340px);
+        width: calc(100% - 340px);
+      }
+      @media only screen and (max-width: 1140px) {
+        .oppia-donation-card-content-wide {
+          margin-bottom: 90px;
+        }
+        .oppia-donate-info {
+          display: block;
+          position: inherit;
+          width: 100%;
+        }
+        .oppia-donate-options {
+          float: clear;
+          width: 100%;
+        }
+      }
+    </style>
```

Comment:
The `<style>` part is already in [donate-page.directive.html](https://github.com/oppia/oppia/blob/develop/core/templates/dev/head/pages/donate-page/donate-page.directive.html)